cli/command/image/client_test.go

```diff
@@ -69,7 +68,8 @@ func (cli *fakeClient) ImagePull(_ context.Context, ref string, options client.I
 	if cli.imagePullFunc != nil {
 		return cli.imagePullFunc(ref, options)
 	}
-	return client.ImagePullResponse{}, nil
+	// FIXME(thaJeztah): how to mock this?
+	return nil, nil
```

Comment:
We need to look at these, as there's no concrete types to use for mocking / testing